packages/nimble-components/src/rich-text/editor/models/create-tiptap-editor.ts

```diff
@@ -210,7 +210,8 @@ function createCustomMentionExtension(
             return [
                 config.viewElement,
                 mergeAttributes(this.options.HTMLAttributes, HTMLAttributes),
-                this.options.renderLabel({
+                // eslint-disable-next-line @typescript-eslint/no-unsafe-call
```

Comment:
Why is there a suppression here?

Reply:
It was a miss. My VS code shows the type error but once reloaded, it doesn't. I have removed the comment.